### DISCUSSION TITLE
actually portable executable (cosmopolitan) support

### DISCUSSION THREAD

**Discussion by cariaso** [pyinstaller/pyinstaller#7039] [Features & Development]
https://github.com/jart/cosmopolitan/issues/141
shows progress on an αcτµαlly pδrταblε εxεcµταblε
https://justine.lol/ape.html
and I'd like to understand if/how this might relate to (or be integrated into) pyinstaller. Can anyone offer insight?

**Comment by bwoodsend** [comment]
Presumably, if Python, every non-pure-Python extension module and PyInstaller's bootloaders were all compiled via cosmopolitan then it would produce an application that would run on all platforms. Assuming that they can even get Python compiled and passing its test suite, I'd be very surprised if it would actually work though. There's much more of a difference between OSs than just the executable format and the libc implementation.
